gettext-gradle-plugin/src/main/groovy/org/mini2Dx/gettext/plugin/task/GeneratePotTask.groovy

```diff
@@ -52,7 +52,7 @@ class GeneratePotTask extends DefaultTask {
             entries.clear();
         }
 
-        final File outputDirectory = new File(project.getBuildDir(), 'gettext');
+        final File outputDirectory = source.outputPath == null ? new File(project.getBuildDir(), 'gettext') : new File(project.getRootProject().getProjectDir(), source.outputPath);
```

Comment:
Tasks should always be relative to current project directory in case of multi-module projects.